recipe/meta.yaml

```diff
@@ -9,36 +9,51 @@ package:
 
 source:
   fn: {{ name }}_{{ version }}.tar.gz
-  url: https://files.salome-platform.org/Salome/other/med-{{ version }}.tar.gz
+  url: https://files.salome-platform.org/Salome/medfile/med-{{ version }}.tar.gz
   sha256: {{ sha256 }}
-    #git_url: https://salsa.debian.org/science-team/med-fichier
-    #git_rev: 8a0b0d21d4bac79772797b0a800cf4d5bf77bc32    
+  patches:
+    - patches/med-4.1.1-check-hdf5-parallel.diff  # [unix]
 
 build:
-  number: 3
-  skip: true  # [py2k or osx]
+  number: 4
+#  string: aster_py{{ CONDA_PY }}_{{mpi}}_np{{CONDA_NPY}}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }} # [mpi != "nompi"]
+#  string: aster_py{{ CONDA_PY }}_seq_np{{CONDA_NPY}}h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}     # [mpi == "nompi"]  
+  skip: true  # [py2k or osx or win or py>=312]
```

Comment:
is there a reason why you decided to drop python 3.12 when it was enabled in a previous pull request?